README.md

```diff
@@ -10,7 +10,7 @@ A Jenkins plugin for automatically forwarding metrics, events, and service check
 
 ### Installation
 
-_This plugin requires Java 8+ and [Jenkins 1.632][2] or newer._
+_This plugin requires [Jenkins 2.164.1][2]._
```

Comment:
Should we add a note that the previous version can still be used for older Jenkins versions?